addons/alerting.libsonnet

```diff
@@ -220,64 +132,19 @@ function(config) {
           equal:
           - alertname
         receivers:
-        - name: Black_Hole
+        %(teamReceivers)s
         - name: Watchdog
-        - name: IDECriticalReceiver
-          %(IDECriticalReceiver)s
-        - name: webappCriticalReceiver
-          %(webappCriticalReceiver)s
-        - name: workspaceCriticalReceiver
-          %(workspaceCriticalReceiver)s
-        - name: platformCriticalReceiver
-          %(platformCriticalReceiver)s
         - name: globalCriticalReceiver
-          %(globalCriticalReceiver)s
-        - name: SlackWarning
-          slack_configs:
-          - send_resolved: true
-            api_url: %(slackWebhookUrlWarning)s
-            channel: '%(slackChannelPrefix)s_warning'
-            title: '[{{ .Status | toUpper }}{{ if eq .Status "firing" }}{{ end }}] %(clusterName)s Monitoring'
-            text: |
-              {{ range .Alerts }}
-              **Please take a look when possible**
-              *Cluster:* {{ .Labels.cluster }}
-              *Alert:* {{ .Labels.alertname }}
-              *Description:* {{ .Annotations.description }}
-              {{ end }}
-            actions:
-            - type: button
-              text: 'Runbook :book:'
-              url: '{{ .CommonAnnotations.runbook_url }}'
-        - name: SlackInfo
-          slack_configs:
+          pagerduty_configs:
           - send_resolved: true
-            api_url: %(slackWebhookUrlInfo)s
-            channel: '%(slackChannelPrefix)s_info'
-            title: '[{{ .Status | toUpper }}{{ if eq .Status "firing" }}{{ end }}] %(clusterName)s Monitoring'
-            text: |
-              {{ range .Alerts }}
-              **No need for human intervention :slightly_smiling_face:
-              *Cluster:* {{ .Labels.cluster }}
-              *Alert:* {{ .Labels.alertname }}
-              *Description:* {{ .Annotations.description }}
-              {{ end }}
-            actions:
-            - type: button
-              text: 'Runbook :book:'
-              url: '{{ .CommonAnnotations.runbook_url }}'
+            routing_key: 'global-pd-routing-key'
```

Comment:
`global-pd-routing-key` shouldn't be hard-coded here 😬 

It is the secret provided by PagerDuty that we'll use to authenticate against our schedule